src/main/kotlin/com/safehill/kclient/models/dtos/websockets/UserConversionManifestDTO.kt

```diff
@@ -0,0 +1,17 @@
+package com.safehill.kclient.models.dtos.websockets
+
+import com.safehill.kclient.models.users.RemoteUser
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class UserConversionManifestDTO(
+    val assetIdsByGroupId: AssetIdsByGroupId,
+    val newUser: RemoteUser,
+    val threadIds: List<String>
+) : WebSocketMessage
+
+
+//todo figure out the type of asset id by group id when dealing with asset and groups.
```

Comment:
By type you mean whether or not is a photo message?

Reply:
The asset descriptor would tell you that. 
You should get latest asset descriptors from remote server for the assets you have to send encryption details

Reply:
By type I was referring to the format of the json for the key "assetIdsByGroupId"... I didn't trigger the request to access to a group of a thread so I had no idea how the json would be formatted for these cases. I was planning to fix the type (model class corresponding to the json format) when I will be dealing with the group access or asset access.